ajax/libs/react-textarea-autosize/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-textarea-autosize",
+  "description": "textarea component for React which grows with content",
+  "version": "4.3.0",
+  "filename": "react-textarea-autosize.min.js",
+  "keywords": [
+    "autosize",
+    "grow",
+    "react",
+    "react-component",
+    "textarea"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/andreypopp/react-textarea-autosize.git"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Andrey Popp",
+    "email": "8mayday@gmail.com"
+  },
+  "homepage": "https://andreypopp.github.io/react-textarea-autosize/",
```

Comment:
@extend1994 as @PeterDaveHello said in https://github.com/cdnjs/cdnjs/pull/11085#issuecomment-297126271
> If it at least links to its GitHub repo with a demo, I'll treat it as homepage @cdnjs/intern3

I think we can this link as the homepage.